### PR TITLE
GA integration

### PR DESCRIPTION
**Pull request by wcjr**

### CODE REVIEW

File: src/views/Requests.svelte [hunk 1]
```diff
@@ -43,6 +56,8 @@ function selectCreator(userId) {
     provider: null,
     size: null,
   })
+
+  filteredRequestsByMine()
```
Comment (forevermatt) [inline src/views/Requests.svelte]: At the moment, we're only calling `selectCreator()` with `$me.id` as the given `userId`, so this works. However, nothing about the `selectCreator()` method is specific to _my_ requests.

I'd recommend changing the `RequestQuickFilter` to call a new `showMyRequests()` method, and have that method call `selectCreator($me.id)` and `filteredRequestsByMine()`.

File: src/views/Requests.svelte [hunk 2]
```diff
@@ -51,28 +66,38 @@ function selectProvider(userId) {
     provider: userId,
     size: null,
   })
+
+  filteredRequestsByProviding()
```
Comment (forevermatt) [inline src/views/Requests.svelte]: As with the `selectCreator()`, the `selectProvider()` isn't limited to selecting requests where _I'm_ the provider, so if the `filteredRequestsByProviding()` GA tracking event is intended to track uses of the "My Commitments" button, then a new `showMyCommitments()` method would probably be worthwhile.

Given both of those changes, the `RequestQuickFilter` would end up looking something like this:
```
<RequestQuickFilter {requestFilter} buttonCssClass="my-1 mx-0"
                    on:all={showAll} on:my-requests={showMyRequests} on:my-commitments={showMyCommitments} />
```

File: src/views/Requests.svelte (hunk 1 above)
Comment (wcjr) [reply src/views/Requests.svelte]: I like that change, I've got some other refactor-type suggestions around filtering I'd like to discuss and group all the related changes into some commits outside of GA.

File: src/views/Requests.svelte (hunk 2 above)
Comment (wcjr) [reply src/views/Requests.svelte]: yeah, I see that too, I think this is the same as my response to `selectCreator` comment.

**Pull request merged**